txstratum/cli.py

```diff
@@ -52,7 +53,22 @@ def execute(args: Namespace) -> None:
 
     # Configure log.
     start_logging()
-    if os.path.exists(args.log_config):
+    if args.json_logs:
```

Comment:
Why don't we use the log config file to enable JSON?

Reply:
Since all I needed in this case was `logging.basicConfig(level=logging.INFO, format='%(message)s')`, I thought it simple enough to stay in the code.

But I can check how to define another logger in the config file and choose which one to use.

Reply:
Actually, now I'm not sure what your suggestion is exactly.

I don't think we could move the configuration I've added to a log config file, since most of what I've added is structlog's configuration. The only thing I would be able to move to a config file is this line: `logging.basicConfig(level=logging.INFO, format='%(message)s')`

Unless I'm missing something.

Reply:
I though about this too, and I think I'm OK using a cli argument instead of using `log.conf` only because it's not that easy to figure out how to configure it. While I believe it is totally possible to use a `log.conf` it's maybe harder than it's worth.

Reply:
> using a cli argument instead of using log.conf

I don't see how the log.conf alone would avoid the need for a cli argument.

My final objective is to give a choice to the user about logging in JSON or not.

So I need to know his choice somehow, because I need to add structlog's processors to configure my JSON logging. That's why I needed the cli argument.

As far as I could find in the docs and googling, it doesn't seem to be possible to add the structlog processors using only a log.conf. Is it possible?

The other way I see is to have another `log.conf`, like a `log_json.conf`. I could do this as well instead of the cli argument (which wouldn't avoid having to add the structlog processors, afaik)

Reply:
I think I understood now. Python's logging module may be completely configurable via `log.conf`, but `structlog.configure` does not load from `log.conf`. An alternative would be extending `log.conf` to support passing structlog configurations, but this seems complicated.